test/app/model/ChessEngine.spec.ts

```diff
@@ -246,7 +246,7 @@ describe('Chess Engine', () => {
       const engine = new ChessEngine(chessboard);
       const movedPiecePosition: Square = { column: 'A', row: 2 };
 
-      const returnedResult = engine.pieceMovesNotCausingAllyKingCheckmate(movedPiecePosition);
+      const returnedResult = engine.pieceMovesNotCausingAllyKingCheck(movedPiecePosition);
```

Comment:
Zmien prosze w koncu na to, zeby tutaj sprawdzalo possibleMoves :) Bo `pieceMovesNotCausingAllyKingCheck ` to nie jest kontrakt naszego Modelu z Presenterem (nie ma tego w ChessModel) i nie potrzebujemy dwóch metod possibleMoves i pieceMovesNotCausingAllyKingCheck, Wszystko idzie przez possibleMoves - tego nie uzywasz poza klasą więc powinno być private.

Reply:
done in next PR